Datadog/IntegrationUnitTests/RUM/RUMViewScopeIntegrationTests.swift

```diff
@@ -0,0 +1,193 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import XCTest
+@testable import DatadogRUM
+@testable import DatadogInternal
+
+final class RUMViewScopeIntegrationTests: XCTestCase {
```

Comment:
suggestion/ Since we're adding an integration test, referring to RUMViewScope—which is an implementation detail—feels out of place. What we're truly testing here is the flow of setting RUM attributes and how they are inherited by different events.

A more descriptive and appropriately scoped name could be:
- `RUMAttributesIntegrationTests`
- `RUMAttributeInheritanceTests`
- `RUMAttributesFlowTests`